cassandra-plugins/src/main/java/co/cask/plugin/etl/realtime/RealtimeCassandraSink.java

```diff
@@ -89,6 +89,17 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
                                 "You must either set both username and password or neither username or password. " +
                                   "Currently, they are username: " + config.username +
                                   " and password: " + config.password);
+    Preconditions.checkArgument(!parseAddresses(config.addresses).isEmpty(),
+                                "At least one pair of IP and port should be provided.",
+                                config.addresses, ADDRESSES_DESCRIPTION);
+  }
+
+  private List<InetSocketAddress> parseAddresses(String addressString) {
+    List<InetSocketAddress> addresses = new ArrayList<>();
+    for (String address : addressString.split(",")) {
+      addresses.add(new InetSocketAddress(address.split(":")[0], Integer.valueOf(address.split(":")[1])));
```

Comment:
split it once and do some verification. For example, check there are exactly 2 parts and make sure we throw useful error messages if the port is not a number